### PR TITLE
Make VirtualBox memory and cpu configurable

### PR DESCRIPTION
**Pull request by nad0u**
Hi !
I'm completely new to Vagrant and I discovered drifter from a friend of mine. I was surprised when I did not find a way in the files (Vagrantfile or parameters.yml) to set the memory and cpus like other parameters. So I made them available as parameters and default the values in case the parameters were not set. I hope this helps.

* This PR is a : Improvement

- [x] ~~Documentation is written~~
- [x] `parameters.yml.dist` is updated
- [x] ~~`playbook.yml.dist` is updated~~
- [x] [Changelog](https://github.com/liip/drifter/blob/master/CHANGELOG.md) was updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ For migration information, you can always have a look at https://liip-drifter.re
 
 ## [Unreleased]
 
+### Added
+
+- Add parameters memory & cpus and changed Vagrantfile and Vagrantfile.dist accordingly
```
Comment (sephii) [inline CHANGELOG.md]: Can you please change the message to something more meaningful to the end user? For example "Allow CPU count and memory to be configured in parameters.yml"

File: parameters.yml.dist [hunk 2]
```diff
@@ -15,6 +15,14 @@ forwarded_ports: {
     "3000": "3000",  # BrowserSync default port
 }
 
+# Virtual machine memory size use in MB. Defaults to 4096. Uncomment and change this
```
Comment (sephii) [inline parameters.yml.dist]: Can you please add a note that these settings are only used with the VirtualBox provider?

**Comment by nad0u** [comment]
Thank you for your reply!

I added a small note for the two parameters and updated the changelog with your proposition (I also noticed that a "Added" section already existed, so I moved the message to the existing one and removed the one I created).

I hope it okay, let me know if further changes need to be made!

**Comment by sephii** [comment]
Congratulations for your first contribution! :tada:

**Pull request merged**